apps/backoffice-v2/src/pages/MerchantMonitoringCreateCheck/components/SwitchesList/SwitchesList.tsx

```diff
@@ -28,7 +28,8 @@ export const SwitchesList: FunctionComponent<{
                       className={`data-[state=checked]:bg-blue-500`}
                       {...field}
                       disabled={option.disabled}
-                      checked={field.value}
+                      // checked={field.value}
```

Comment:
Leftover?

Reply:
Uncommented and changed the field's `name` to include the label in case of duplicate names.